include/deal.II/fe/mapping.h

```diff
@@ -488,23 +485,24 @@ class Mapping : public Subscriptor
 
 protected:
   /**
-   * Indicate fields to be updated in the constructor of FEValues. Especially,
-   * fields not asked for by FEValues, but computed for efficiency reasons
-   * will be notified here.
+   * Given a set of update flags, compute which other quantities <i>also</i>
+   * need to be computed in order to satisfy the request by the given flags.
+   * Then return the combination of the original set of flags and those
+   * just computed.
    *
-   * See
-   * @ref UpdateFlagsEssay.
-   */
-  virtual UpdateFlags update_once (const UpdateFlags) const = 0;
-
-  /**
-   * The same as update_once(), but for the flags to be updated for each grid
-   * cell.
+   * As an example, if @p update_flags contains update_JxW_values (i.e., the
+   * product of the determinant of the Jacobian and the weights provided by
+   * the quadrature formula), most mappings will require the computation of
+   * the full Jacobian matrix in order to compute its determinant. They
+   * would then return not just update_JxW_values, but also
+   * update_jacobians.
```

Comment:
Good point. I'd rather not bring up the Hessian example because that works on the finite element side, whereas the example above lives on the mapping side. I've replaced the text above by the following:
```
   * As an example, if @p update_flags contains update_JxW_values                                                                                            
   * (i.e., the product of the determinant of the Jacobian and the                                                                                           
   * weights provided by the quadrature formula), a mapping may                                                                                              
   * require the computation of the full Jacobian matrix in order to                                                                                         
   * compute its determinant. They would then return not just                                                                                                
   * update_JxW_values, but also update_jacobians. (This is not how it                                                                                       
   * is actually done internally in the derived classes that compute                                                                                         
   * the JxW values -- they set update_contravariant_transformation                                                                                          
   * instead, from which the determinant can also be computed -- but                                                                                         
   * this does not take away from the instructiveness of the example.)                                                                                       
```